### PR TITLE
fix typos and removing couple typescript `as any`

### PR DESCRIPTION
**Pull request by pikax**
Typos and remove unnecessary `as any`

### CODE REVIEW

**Comment by liximomo** [comment]
@pikax Can you move the "removing `as any`" things to a separate PR?

**Comment by pikax** [comment]
done

File: src/setup.ts [hunk 1]
```diff
@@ -146,16 +146,18 @@ export function mixin(Vue: VueConstructor) {
     if (!binding) return;
 
     if (isFunction(binding)) {
+      // keep typescript happy with the binding type.
```
Comment (liximomo) [inline src/setup.ts]: You missed here.

Comment (pikax) [reply src/setup.ts]: fixed it, thanks :)

**Pull request merged**